### ISSUE TITLE
Froms example - add/edit array of elephant or lion.

### ISSUE THREAD

**Issue by murraybauer** [angular-redux/example-app#5]
Great example app!  

Would be interested to see the example include of how angular redux forms would work when adding or editing an elephant or lion with a form to the existing state array.

I am trying to understand the benefits using redux for forms (in a project that already uses redux for state management).  The feedback form example with a default reducer is quite basic and maybe most would just still use a generic angular form and access the .valueChanges observable of a particular form control to compute the characters remaining.

**Comment by SethDavenport** [comment]
Yeah the example is new and needs some beefing up. PRs welcome :)

As for the value of @angular-redux/forms over the basic `.valueChanges` stuff I guess it depends on your use case.

I have found that managing complex reactive logic with very large forms gets cumbersome quickly. Once you are deriving behaviour from a bunch of fields simultaneously things can get a bit hairy especially to debug. Personally I find that if it's all in the store I can use DevTools to help me inspect, rewind etc. just as I would with any other app state.

I also personally find the built-in form builder stuff a bit weird to use, and this lets me avoid a lot of it by building validations etc in a more framework-agnostic way. But this is I admit a matter of taste.

Tl;Dr I would say use this if you want to be able to use Redux DevTools on your form state.
